xbmc/guilib/GUIFontTTF.h

```diff
@@ -119,8 +134,8 @@ class CGUIFontTTF
 
   struct RunInfo
   {
-    int startOffset;
-    int endOffset;
+    unsigned int startOffset;
```

Comment:
```suggestion
    unsigned int m_startOffset;
```

Reply:
Has done on some of the the different structure values, hope Windows becomes happy.